src/Integration.Vsix/SonarLintTagger/IssueToIssueMarkerConverter.cs

```diff
@@ -58,7 +62,8 @@ public IssueMarker Convert(IAnalysisIssue issue, ITextSnapshot textSnapshot)
 
             var span = issueSpanCalculator.CalculateSpan(issue, textSnapshot);
 
-            return span == null ? null : new IssueMarker(issue, span.Value);
+            var issueViz = issueVizConverter.Convert(issue);
```

Comment:
This converter is called when the issue consumer processes analysis results.
Previously, it was calculating the span for each issue at this point. Now it's calculating the span and converting the issue into an issue viz.
I'll combine the converters into a single type later.